mlflow/tracking/client.py

```diff
@@ -259,6 +259,9 @@ def log_artifacts(self, run_id, local_dir, artifact_path=None):
         """
         self._tracking_client.log_artifacts(run_id, local_dir, artifact_path)
 
+    def record_logged_model(self, run_id, mlflow_model):
```

Comment:
Let's add a docstring/annotate this as experimental?

Reply:
Yeah I think it's safer to make it private.